packages/perseus/src/__tests__/__snapshots__/renderer.test.tsx.snap

```diff
@@ -230,7 +230,7 @@ exports[`renderer rendering should render block math on mobile 1`] = `
           style="overflow-x: auto; overflow-y: hidden; margin-top: -10px; margin-bottom: -10px; transform: translate3d(0,0,0); padding: 10px 16px 10px 16px;"
         >
           <span
-            style="display: block; width: 100%; transform: scale(1, 1)  translate(0, 8px); transform-origin: 0 0; opacity: 0;"
+            style="display: block; width: 100%; transform: scale(1, 1); transform-origin: 0 0; opacity: 1;"
```

Comment:
As long as we're changing this line, could we get rid the `transform` and `transform-origin` properties? Do they have any effect now?